src/routes/signup.tsx

```diff
@@ -49,6 +49,8 @@ const SignUpForm = () => {
   const realms = usePromise(getRealms) || []
   const emitError = useErrorEmitter()
 
+  realms.sort((a, b) => a.name.localeCompare(b.name))
```

Comment:
Let's avoid running this sort on every render. It is against the rules of hooks to modify variables directly in the component body: https://react.dev/reference/rules/components-and-hooks-must-be-pure.

Breaking the rules of hooks doesn't mean that your code won't work, but it does mean that future changes to react/preact might break your code even if it works now.

Since `sort()` mutates the underlying variable, probably better to do chain it on to the promies so it just happens once:

```
usePromise(() => getRealms().then(realms => realms.sort(...)) || []
```

You could also modify getRealms function directly so that it always returns a sorted list. Probably would be a worthwhile change.